asiana21 DOCSUP-658 (#93)
BayoNet/ClickHouse: docs/en/query_language/functions/ym_dict_functions.md

For information about creating reference lists, see the section "Dictionaries".

> ## Multiple geobases {#multiplegeobases}

Reviewer: ```suggestion
## Multiple Geobases {#multiplegeobases}
```

Converts a region to a continent. In every other way, this function is the same as 'regionToCity'.
Example: `regionToContinent(toUInt32(213)) = 10001` converts Moscow (213) to Eurasia (10001).

### regionToTopContinent (#regiontotopcontinent)

Finds the highest continent in the hierarchy for the region.

**Syntax**

```sql
regionToTopContinent(id[, geobase]);
```

**Parameters**

> - `id` —  the region ID from the Yandex geobase. [UInt32](../../data_types/int_uint.md).

Reviewer: ```suggestion
- `id` —  Region ID from the Yandex geobase. [UInt32](../../data_types/int_uint.md).
```

> - `geobase` — Optional. The dictionary key that is used in [multiple geobases](#multiplegeobases). [String](../../data_types/string.md).

Reviewer: ```suggestion
- `geobase` —  Dictionary key. See [Multiple Geobases](#multiplegeobases). [String](../../data_types/string.md). Optional.
```

**Returned value(s)**

> - The continent (the latter when you climb the hierarchy of regions), including it.

Reviewer: ```suggestion
- Identifier of the top level continent (the latter when you climb the hierarchy of regions).
```
